soh/src/overlays/actors/ovl_En_GirlA/z_en_girla.c

```diff
@@ -452,11 +452,6 @@ void EnGirlA_Init(Actor* thisx, PlayState* play) {
 }
 
 void EnGirlA_Destroy(Actor* thisx, PlayState* play) {
-    EnGirlA* this = (EnGirlA*)thisx;
-
-    if (this->isInitialized) {
-        SkelAnime_Free(&this->skelAnime, play);
-    }
```

Comment:
just noting that this was mentioned already so it's easy to see when scrolling through
> En_GirlA Does not initialize a skeleton, so I removed the SkelAnime_Free